backgrounds/meson.build

```diff
@@ -70,3 +71,8 @@ configure_file(
     configuration: cdata,
     install_dir: mate_background_dir,
 )
+
+# PLASMA Wallpaper definition
+plasma_background_dir = join_paths(path_datadir, 'backgrounds', 'solus')
```

Comment:
Not sure what you were trying to accomplish here but it isn't necessary. I'm aware that Plasma has a special location, we already do a symlink in the package itself.